Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:8
+
+ENV PATH /usr/src/app/front-end/node_modules/.bin:$PATH
+
+WORKDIR /usr/src/app
+COPY . .
+RUN yarn --production
```

Comment:
Don't we need to `yarn install` first? :thinking: Or is this what `yarn --production` does? :P

Reply:
installs with only prod dependencies as far as i understand

Reply:
I think my problems came from running yarn --prod and then not having react scripts to do the yarn build command, a normal yarn install would probably solve both problems since unneeded dependencies would not be included when building the bundle anyway